libphal/phal_dump.C

```diff
@@ -131,6 +131,30 @@ struct pdbg_target* getProcFromFailingId(const uint32_t failingUnit)
 	return proc;
 }
 
+/**
+ * @brief Function will write reason code and recovery action in a file
+ * @param[in] dumpPath Path of the file
+ * @param[in] reasonCode Reason code of sbe failure
+ * @param[in] recoveryAction Reovery action code for sbe failure
+ *
+ * @return NULL
+ *
+ */
+void writeSbeData(const std::filesystem::path& dumpPath, uint32_t reasonCode,
+                  uint32_t recoveryAction)
```

Comment:
is recoveryAction uint32_t or uint64_t? and also reasonCode ?

Reply:
its unint32_t